test/unit/Site.test.js

```diff
@@ -91,10 +87,10 @@ test('Site Generate builds the correct amount of assets', async () => {
   expect(fs.existsSync(path.resolve('inner/_site/markbind/fontawesome/webfonts/font2.ttf'))).toEqual(true);
 
   // layouts
-  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/footer.md'))).toEqual(true);
-  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/head.md'))).toEqual(true);
-  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/header.md'))).toEqual(true);
-  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/navigation.md'))).toEqual(true);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/footer.md'))).toEqual(false);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/head.md'))).toEqual(false);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/header.md'))).toEqual(false);
+  expect(fs.existsSync(path.resolve('inner/_site/markbind/layouts/default/navigation.md'))).toEqual(false);
```

Comment:
I believe that these four lines can just be removed, since `const expectedNumBuilt = 16;` should already provide some assurance that these files cannot exist.